UnrealPlugin/README.md

```diff
@@ -1,37 +1,113 @@
-# Playfab GSDK Unreal Plugin Preview
+# Instructions on Integrating the Playfab GSDK Unreal Plugin
 This Unreal Plugin is implementing the GSDK directly in Unreal Engine.
 
-It was tested with Unreal Engine 4.26.2, but should work with other engines as well.
+It was tested with Unreal Engine 4.26.2, but should work with other engines as well. The sample game these instructions were created with was called ThirdPersonMP, so replace anywhere you see that with your game name.
+
+# Prerequisites:
+- Download Unreal Engine Source Build by following [these instructions](https://docs.unrealengine.com/4.26/en-US/ProgrammingAndScripting/ProgrammingWithCPP/DownloadingSourceCode/) from the Unreal website
+- Download Unreal Engine (this was tested on 4.26.2)
+- Download the [Unreal PlayfabGSDK Plugin folder](https://github.com/PlayFab/gsdk/tree/master/UnrealPlugin)
+- Download Visual Studio
```

Comment:
you can add a link about the free community version here https://visualstudio.microsoft.com/vs/community/